cmd/cmd_controller.js

```diff
@@ -279,6 +279,7 @@ class EmbarkController {
             engine.startService("storage");
             engine.startService("codeGenerator");
             engine.startService("webServer");
+            engine.startService("namingSystem");
```

Comment:
Naming System wasn't enabled in console

Reply:
I know the IPC wasn't started in the console either. Was it added?

Reply:
I think namingSystem and console command were added at the same time, that is why it wasn't there.
I have added it so we can interact with ENS via console

Reply:
I have touch IPC